### PR TITLE
upddep scrapelib

### LINKED ISSUE #213: AttributeError: 'Scraper' object has no attribute 'urlopen'
I am very excited about this project, and would like to contribute what I can-as long as I can get it working! :)

I created a brand new Ubuntu 16.04 desktop VM just for this project, and cloned the repos into /opt. 

I have met all the requirements. One of them was for scrapelib, and I put it into it's own folder under /opt from https://github.com/jamesturk/scrapelib.git (/opt/scrapelib).
To get the ./run script to run, I had to point to the scrapelib folder/module/library. I did this after seeing:

```
politic@politic-VirtualBox:/opt/congress$ ./run fdsys --collections=BILLSTATUS
Traceback (most recent call last):
  File "./run", line 54, in <module>
    import utils
  File "/opt/congress/tasks/utils.py", line 15, in <module>
    import scrapelib
ImportError: No module named scrapelib
```

...and I did this by modifying /opt/congress/tasks/utils.py thusly:

```
import os
import os.path
import errno
import sys

# added for scrapelib
sys.path.insert(0, "/opt/scrapelib")
[...]
```

This changed my output (same command as above) to:

```
politic@politic-VirtualBox:/opt/congress$ sudo ./run fdsys --collections=BILLSTATUS
Downloading: https://www.gpo.gov/smap/fdsys/sitemap.xml
Traceback (most recent call last):

  File "./run", line 71, in <module>
    task_mod.run(options)

  File "/opt/congress/tasks/fdsys.py", line 92, in run
    update_sitemap_cache(options, listing)

  File "/opt/congress/tasks/fdsys.py", line 113, in update_sitemap_cache
    update_sitemap(fdsys_baseurl + "fdsys/sitemap.xml", None, [], options, listing)

  File "/opt/congress/tasks/fdsys.py", line 156, in update_sitemap
    'binary': True

  File "/opt/congress/tasks/utils.py", line 319, in download
    response = scraper.urlopen(url, **urlopen_kwargs)

AttributeError: 'Scraper' object has no attribute 'urlopen'
```

I program in lots of languages on linux & windows (including Perl), but I've always kept python at arm's distance since it was created. With this project I will need to brush up, it seems! 

To the owner: I know this isn't a bug, but it looks like this is not for bug reports(?). If this is not the correct forum to get the project going, please forgive me.

TIA!

P.S. The portion of code pointed to above is:

```
[...]
# Download from the network and cache to disk.
    else:
        try:
            logging.info("Downloading: %s" % url)

            if postdata:
                response = scraper.urlopen(url, 'POST', postdata, **urlopen_kwargs)
            else:
                if not needs_content:
                    mkdir_p(os.path.dirname(cache_path))
                    scraper.urlretrieve(url, cache_path, **urlopen_kwargs)
                    return True

                response = scraper.urlopen(url, **urlopen_kwargs)
[...]
```
P.S.S. It wasn't mentioned, but running the test (./test/run) after just following instructions/requirements.txt yielded:

```
politic@politic-VirtualBox:/opt/congress$ ./test/run
WARNING:root:Using original YAML file...
E.................................
======================================================================
ERROR: test_hearing (test_house_hearing.HearingInfo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/congress/test/test_house_hearing.py", line 15, in test_hearing
    "test/fixtures/committees-current.yaml"):
  File "tasks/utils.py", line 689, in yaml_load
    yaml_data = direct_yaml_load(filename)
  File "tasks/utils.py", line 601, in direct_yaml_load
    import yaml
ImportError: No module named yaml

----------------------------------------------------------------------
Ran 34 tests in 0.017s

FAILED (errors=1)
```

...so I installed python-yaml:

```
politic@politic-VirtualBox:/opt/congress$ sudo apt install python-yaml
```

...and the test ran OK:

```
politic@politic-VirtualBox:/opt/congress$ ./test/run
WARNING:root:Using original YAML file...
..................................
----------------------------------------------------------------------
Ran 34 tests in 0.117s

OK
```

### PR DESCRIPTION
**Pull request by acxz**
This PR resolves #213 

I have made changes to how we use `scrapelib` based on the updated API of the 1.x versions.
The following [migration guide](https://scrapelib.readthedocs.io/en/latest/migration.html) was used as reference.

### CODE REVIEW

File: tasks/utils.py [hunk 1]
```diff
@@ -306,21 +306,21 @@ def download(url, destination=None, options={}):
             logging.info("Downloading: %s" % url)
 
             if postdata:
-                response = scraper.urlopen(url, 'POST', postdata, **urlopen_kwargs)
+                response.post = scraper.urlopen(url, postdata, **urlopen_kwargs)
```
Comment (JoshData) [inline tasks/utils.py:309]: This line doesn't look right.

Comment (acxz) [reply tasks/utils.py:309]: Thanks for the review!

Ah yes mb let me fix it rq.

Comment (acxz) [reply tasks/utils.py]: @JoshData fixed

**Comment by JoshData** [comment]
Thanks! I will try this out when I get a chance.

**Comment by acxz** [comment]
@JoshData Its been more than a week can you provide closure to this PR?

**Comment by JoshData** [comment]
I'm testing it now. Gimme two more days.

**Comment by JoshData** [comment]
Thank you for the changes.

**Pull request merged**